backend/algorithms/errors.json

```diff
@@ -11666,73 +11170,18 @@
         ],
         "broke at": "Index 2, ."
     },
-    "FOOD3010": {
-        "condition": {
-            "original": "Pre-requisites:FOOD1120,FOOD2320,CHEM2921<br/><br/>",
-            "processed": "FOOD1120,FOOD2320,CHEM2921"
-        },
-        "tokens": [
-            "(",
-            "FOOD1120,FOOD2320,CHEM2921",
-            ")"
-        ],
-        "broke at": "Index 1, FOOD1120,FOOD2320,CHEM2921"
-    },
-    "FOOD3020": {
-        "condition": {
-            "original": "Pre-Requisites:FOOD1120, CHEM2921<br/>Co-requisite: FOOD3010<br/><br/>",
-            "processed": "FOOD1120, CHEM2921 [FOOD3010]"
-        },
-        "tokens": [
-            "(",
-            "FOOD1120,",
-            "CHEM2921",
-            "[",
-            "FOOD3010",
-            "]",
-            ")"
-        ],
-        "broke at": "Index 1, FOOD1120,"
-    },
-    "FOOD3030": {
-        "condition": {
-            "original": "Prerequisites: FOOD2320, FOOD3010<br/><br/>",
-            "processed": "FOOD2320, FOOD3010"
-        },
-        "tokens": [
-            "(",
-            "FOOD2320,",
-            "FOOD3010",
-            ")"
-        ],
-        "broke at": "Index 1, FOOD2320,"
-    },
-    "FOOD3060": {
-        "condition": {
-            "original": "Prerequisites: PHYS1111, MATH1031, MATH1041<br/><br/>",
-            "processed": "PHYS1111, MATH1031, MATH1041"
-        },
-        "tokens": [
-            "(",
-            "PHYS1111,",
-            "MATH1031,",
-            "MATH1041",
-            ")"
-        ],
-        "broke at": "Index 1, PHYS1111,"
-    },
     "FOOD6804": {
         "condition": {
             "original": "Prerequsite :CHEM3811or INDC2003<br/><br/>",
-            "processed": ":CHEM3811or INDC2003"
+            "processed": "CHEM3811or INDC2003"
```

Comment:
This one's just silly but it's good to see that the new prerequisite regex works as intended